README.md

```diff
@@ -83,6 +86,15 @@ To fire an opened ligtbox event, simply call the `openedLightbox` method of the
 photo.openedLightbox(context);
 ```
 
+#### Action Clicked
+To fire an action clicked event, simply call the `actionClicked` method of the PXLPhoto that the action click is being driven from, and an "Action Clicked" event will be fired containing all of the necessary analytics information.
```

Comment:
Can you adjust the text here, they need to pass the product link of the product clicked for example if a photo has multiple products.

Reply:
yes, great catch!